index.js

```diff
@@ -138,13 +138,26 @@ var Node = function (path, opts, disposable) {
     daemonPid: function () {
       return this.subprocess && this.subprocess.pid
     },
-    getConf: function (key, cb) {
+    getConfig: function (key, cb) {
+      if (typeof key === 'function') {
+        cb = key
+        key = ''
+      }
       var t = this
       var result = ''
       run(IPFS_EXEC, ['config', key], {env: t.env})
         .on('error', cb)
         .on('data', function (data) { result += data })
-        .on('end', function () { cb(null, result.trim()) })
+        .on('end', function () {
+          cb(null, result.trim())
+        })
+    },
+    setConfig: function (key, value, cb) {
+      var t = this
```

Comment:
Hm, `package.json` doesn't contain any node version (somebody should open an issue about that) so not sure what compatibility we're reaching for. But with that said, if we were using ES2015 (new name for ES6, yay :( ), this assignment wouldn't have been there since the functions below would have been [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).